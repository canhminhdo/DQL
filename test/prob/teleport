#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < teleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/teleport.out
diff $DIR/teleport.expected $DIR/teleport.out
rm $DIR/teleport.out