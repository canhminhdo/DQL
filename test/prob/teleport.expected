==========================================
reduce in TELEPORT : initQState |= [H(1) ; CX(1,2) ; CX(0,1) ; H(0) ; if P(1,
    |0>) then skip else X(2) fi ; if P(0,|0>) then skip else Z(2) fi](P(2,
    qubitAt(initQState, 0)) with prob == 1) .
result JudgmentSet: emptyJS
Bye.
